Neos.ContentRepository/src/Feature/NodeRemoval/Event/NodeAggregateCoverageWasRestored.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Feature\NodeRemoval\Event;
+
+use Neos\ContentRepository\DimensionSpace\DimensionSpace\DimensionSpacePointSet;
+use Neos\ContentRepository\SharedModel\Node\OriginDimensionSpacePoint;
+use Neos\ContentRepository\SharedModel\Workspace\ContentStreamIdentifier;
+use Neos\ContentRepository\SharedModel\Node\NodeAggregateIdentifier;
+use Neos\ContentRepository\Feature\Common\EmbedsContentStreamAndNodeAggregateIdentifier;
+use Neos\ContentRepository\Feature\Common\PublishableToOtherContentStreamsInterface;
+use Neos\ContentRepository\SharedModel\User\UserIdentifier;
+use Neos\EventSourcing\Event\DomainEventInterface;
+use Neos\Flow\Annotations as Flow;
+
+#[Flow\Proxy(false)]
+final class NodeAggregateCoverageWasRestored implements
```

Comment:
add some comment linking to the command; or alternatively add again description what the different DSPs mean ;)